crates/xilem_masonry/examples/mason.rs

```diff
@@ -20,6 +20,9 @@ fn app_logic(data: &mut AppData) -> impl MasonryView<AppData> {
         .collect::<Vec<_>>();
     flex((
         button(label, |data: &mut AppData| data.count += 1),
+        checkbox("Check me", data.active, |data: &mut AppData, checked| {
```

Comment:
Thanks for not removing the activate/deactivate buttons; they're still useful for debugging [#masonry > Hot state for new widgets](https://xi.zulipchat.com/#narrow/stream/317477-masonry/topic/Hot.20state.20for.20new.20widgets)